configure

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+SRC_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+NGINX_VERSION=${NGINX_VERSION:-1.13.3}
+BUILD_INFO_FILE="$SRC_DIR/.build.info"; test -f "$BUILD_INFO_FILE" && . "$BUILD_INFO_FILE"
+CONFIG_OPTS=($CONFIG_OPTS --with-compat --add-dynamic-module=$SRC_DIR)
+
+if [[ -z "$NGINX_SRC_DIR" ]]; then
+    read -t 60 -e -p "Path to NGINX (leave blank to download version $NGINX_VERSION): " NGINX_SRC_DIR ||:
+
+    if [[ -z "$NGINX_SRC_DIR" ]]; then
+        NGINX_SRC_DIR="$SRC_DIR/nginx-$NGINX_VERSION"
+
+        # Double check that the directory doesn't already exist before downloading NGINX anew 
+        if [[ ! -d "$NGINX_SRC_DIR" ]]; then
+            wget http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz
+            tar -xzf nginx-${NGINX_VERSION}.tar.gz
+        fi
+    fi
+fi
+
+if [[ -z "${NGINX_DEBUG+xxx}" ]]; then
+    read -t 10 -p "Do you want to enable debug features (not recommended for production usage) [y/N]: " NGINX_DEBUG ||:
+fi    
+
+if [[ "$NGINX_DEBUG" =~ ^([yY][eE][sS]|[yY])+$ ]]; then
+    CONFIG_OPTS=($CONFIG_OPTS --with-debug --with-cc-opt="-O0 -g")       
```

Comment:
> Change `$CONFIG_OPTS` to `${CONFIG_OPTS}`

Why? Style only? I prefer the format without the extra braces when the braces aren't required because it reduces the noise.